lib/ivis_opengl/pielighting.h

```diff
@@ -0,0 +1,103 @@
+/*
+	This file is part of Warzone 2100.
+	Copyright (C) 1999-2004  Eidos Interactive
+	Copyright (C) 2005-2020  Warzone 2100 Project
```

Comment:
```suggestion
	Copyright (C) 2005-present  Warzone 2100 Project
```

Reply:
We want to use the year, so this should be 2005-2024

Reply:
OK, if this is the current policy, let's leave it as is.